### DISCUSSION TITLE
Important information for collection maintainers

### DISCUSSION THREAD

**Discussion by Andersson007** [netbox-community/ansible_modules#711] [General]
##### SUMMARY
Dear maintainers,

This is important for your collections!

- In accordance with the [Community decision](https://github.com/ansible-community/community-topics/issues/51), we have created the [news-for-maintainers](https://github.com/ansible-collections/news-for-maintainers) repository for announcements of changes impacting collection maintainers (see the [examples](https://github.com/ansible-collections/news-for-maintainers#examples-of-news-posted-via-this-repository)) instead of [Issue 45](https://github.com/ansible-collections/overview/issues/45) that will be closed soon.
  - To keep yourself well-informed and, therefore, things in your collection working, please subscribe to the [repository](https://github.com/ansible-collections/news-for-maintainers) by using the `Watch` button in the upper right corner on the repository's home page.
  - If you do not want to get notifications about related discussions, please subscribe only to `Issues`.
  - Please read the brief [guidelines](https://github.com/ansible-collections/news-for-maintainers#guidelines) on how the repository should be used.
  - Please avoid unnecessary discussions in issues, use the [Discussions](https://github.com/ansible-collections/news-for-maintainers/discussions) feature. Every comment posted will notify **a lot** of folks!
 
- Also we would like to remind you about the [Bullhorn](https://github.com/ansible/community/wiki/News#the-bullhorn) contributor newsletter which has recently started to be released weekly. To learn what it looks like, see the [past releases](https://github.com/ansible/community/wiki/News#the-bullhorn). Please [subscribe](http://eepurl.com/gZmiEP) and [talk](https://github.com/ansible/community/issues/546#issuecomment-973381766) to the Community via Bullhorn!
 
- Join us in [#ansible-social](https://matrix.to/#/#social:ansible.im) (for news reporting & chat), [#ansible-community](https://matrix.to/#/#community:ansible.im) (for discussing collection & maintainer topics), and other channels on [Matrix/IRC](https://docs.ansible.com/ansible/latest/community/communication.html#real-time-chat).

- Help the Community and the Steering Committee to make right decisions by taking part in discussing and voting on the [Community Topics](https://github.com/ansible-community/community-topics/issues) that impact the whole project and the collections in particular. Your opinion there will be much appreciated!
 
Thank you!

**Comment by ryanmerolle** [comment]
Move to a discussion since there are no actions for this project at the time.  THanks @Andersson007
